samples/kubernetes-event-source/README.md

```diff
@@ -25,8 +25,8 @@ kubectl -n default apply -f samples/kubernetes-event-source/channel.yaml
 
 ### Service Account
 
-1. Create a Service Account that the `Receive Adapter` runs as. The
-   `Receive Adapater` watches for Kubernetes events and forwards them to the
+1. Create a Service Account that the _Receive Adapter_ runs as. The
+   _Receive Adapter_ watches for Kubernetes events and forwards them to the
    Knative Eventing Framework. If you want to re-use an existing Service Account
    with the appropriate permissions, you need to modify the
```

Comment:
The sentence just stops. Should it end with "... yaml file accordingly"?

Reply:
Thanks for noticing this @sjwoodman. I decided to just remove this sentence since re-using an existing service account complicates the instructions for multiple steps and that complication doesn't seem worth it.